notebooks/text_classification/text_classification_classical_text_explainer.ipynb

```diff
@@ -10,6 +10,7 @@
   },
   {
    "cell_type": "code",
+   "execution_count": null,
```

Comment:
nit: I think that can be removed?

Reply:
Hmm - when the notebook is loaded it shows a pop saying validation for the notebook failed because it expects the "execution_count" for a code cell. Hence added that for better user experience!